bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html

```diff
@@ -10,10 +10,10 @@
 
 {% block article %}
 <h1 class="mzp-c-article-title">{{ _('Mozilla Community Participation Guidelines') }}</h1>
-{% if l10n_has_tag('participation_nov2018') %}
-  <h4>{{ _('Version 3.0 – Updated November 28, 2018') }}</h4>
+{% if l10n_has_tag('participation_jan2020') %}
+  <h4>{{ _('Version 3.1 – Updated January 15, 2020') }}</h4>
```

Comment:
Note: we'll need to change this date to the correct publishing date once we have final signoff.